### ISSUE TITLE
Return to news feed or access menu after entering an issue?

### ISSUE THREAD

**Issue by mxmilkb** [slapperwan/gh4a#945]
The only major UI frustration with this app currentlly.

Maybe a long press on the back button could return to the news feed?

Being able to swipe from the left to get the many at any point would be also be handy.

**Comment by Mrfiregem** [comment]
Yeah, it's not very intuitive to have to go back multiple times to get back to the feed you started in
